clients/venice-client/src/main/java/com/linkedin/venice/fastclient/ClientConfig.java

```diff
@@ -283,14 +296,20 @@ public ClusterStats getClusterStats() {
     private long routingUnavailableRequestCounterResetDelayMS = -1;
     private int routingPendingRequestCounterInstanceBlockThreshold = -1;
 
+    // TODO: 2 was set initially for singleGet based multiGet for a specific use case.
+    // Now as streaming batchGet is validated, this number should be increased further.
+    // Check how thinclient handles this: Global setting or per store setting.
+    // Setting this in fastclient which uses this in the client itself vs thinclient
+    // which uses this in router should be considered before setting this default limit.
+    // TODO: Add testcases once this is finalized.
     private int maxAllowedKeyCntInBatchGetReq = 2;
 
     private DaVinciClient<StoreMetaKey, StoreMetaValue> daVinciClientForMetaStore;
 
     private long metadataRefreshIntervalInSeconds = -1;
 
     private boolean longTailRetryEnabledForSingleGet = false;
-    private int longTailRetryThresholdForSingletGetInMicroSeconds = 1000; // 1ms.
+    private int longTailRetryThresholdForSingleGetInMicroSeconds = 1000; // 1ms.
 
     private boolean longTailRetryEnabledForBatchGet = false;
     private int longTailRetryThresholdForBatchtGetInMicroSeconds = 10000; // 10ms.
```

Comment:
Thanks for fixing other typos, and this one also a typo:
longTailRetryThresholdFor**Batcht**GetInMicroSeconds
Can you fix it as well?

Reply:
Fixed it.